CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## [2.0](https://github.com/arborx/arborx/tree/v2.0) (2025-04-22)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.7...v2.0)
+
+**New features:**
+- New interfaces for search indexes (APIv2) [\#1155](https://github.com/arborx/ArborX/pull/1155), [\#1157](https://github.com/arborx/ArborX/pull/1157), [\#1161](https://github.com/arborx/ArborX/pull/1161)
+- Hyper-dimensional geometries  [\#1100](https://github.com/arborx/ArborX/pull/1100), [\#1148](https://github.com/arborx/ArborX/pull/1148), [\#1146](https://github.com/arborx/ArborX/pull/1146), [\#1154](https://github.com/arborx/ArborX/pull/1154)
+
+**Build Changes:**
+- Require C++20 [\#1215](https://github.com/arborx/ArborX/pull/1215)
+- Require Kokkos 4.5.00 [\#1233](https://github.com/arborx/ArborX/pull/1233)
+- Require CMake 3.22 [\#1188](https://github.com/arborx/ArborX/pull/1188)
+- Update compiler requirements for C++20 [\#1215](https://github.com/arborx/ArborX/pull/1215)
+
+**Enhancements:**
+- Promote `ArborX::Triangle` from the EXPERIMENTAL namespace [\#1154](https://github.com/arborx/ArborX/pull/1154)
+- Add `Segment` geometry [EXPERIMENTAL] [\#1165](https://github.com/arborx/ArborX/pull/1065)
+- cmake: make install path configurable [\#1174](https://github.com/arborx/ArborX/pull/1174)
+- Add `DefaultIndexableGetter` [EXPERIMENTAL] [\#1181](https://github.com/arborx/ArborX/pull/1181)
+- Add `ARBORX_VERSION` macro [\#1190](https://github.com/arborx/ArborX/pull/1190)
+- Allow running DBSCAN in different precision [\#1203](https://github.com/arborx/ArborX/pull/1203)
+- Add an example using custom distance metric for nearest search [\#1115](https://github.com/arborx/ArborX/pull/1115)
+- Move `MinimumSpanningTree` to `Experimental` namespace [\#1213](https://github.com/arborx/ArborX/pull/1213)
+- Add Euclidean MST example [\#1214](https://github.com/arborx/ArborX/pull/1214)
```

Comment:
Do we reallyy want examples mentioned in the changelog?

Reply:
I'm not 100% certain.

Reply:
I see how if we don;t have much to list we might fill compelled to add, but frankly I think we should keep them crisp.  When looking at a dependency changelog, this is not something I would care about.